packages/protobuf-test/src/create.test.ts

```diff
@@ -502,6 +515,21 @@ describe("create()", () => {
         expect(msg.mapInt32WrappedUint32Field).toStrictEqual({});
         expect(hasOwn("mapInt32WrappedUint32Field")).toBe(true);
       });
+      test("with custom prototype", () => {
+        const msg = create(desc);
+        const hasCustomPrototype =
+          Object.getPrototypeOf(msg) !== Object.prototype;
+        expect(hasCustomPrototype).toBe(true);
+      });
+    });
+    describe("from edition2023 with proto3 features", () => {
+      const desc = test_messages_proto3_editions_ts.TestAllTypesProto3Schema;
+      test("without custom prototype", () => {
+        const msg = create(desc);
+        const hasCustomPrototype =
+          Object.getPrototypeOf(msg) !== Object.prototype;
+        expect(hasCustomPrototype).toBe(false);
+      });
```

Comment:
We're using [editions/golden/test_messages_proto3_editions.proto](https://github.com/protocolbuffers/protobuf/blob/v27.1/editions/golden/test_messages_proto3_editions.proto) to test that an editions file with file option `features.field_presence = IMPLICIT` does not use the prototype chain.